packages/botonic-plugin-contentful/src/contentful/manage/manage-contentful.ts

```diff
@@ -75,9 +75,10 @@ export class ManageContentful implements ManageCms {
       )
     }
     if (!context.allowOverwrites) {
-      if (entry.fields[field.cmsName][context.locale]) {
+      const value = entry.fields[field.cmsName][context.locale]
+      if (value) {
         throw new CmsException(
-          `Cannot overwrite field ${field.cmsName} of entry ${entry.sys.id} because ManageContext.allowOverwrites is enabled`
+          `Cannot overwrite field $'{field.cmsName}' of entry '${entry.sys.id}' (has value '${value}) because ManageContext.allowOverwrites is false`
```

Comment:
Shouldn't the quote be around the dollar sign here? `$'{field.cmsName}' `

Reply:
:+1: good eyes!